job-descriptions/forward-deployed-engineer.md

```diff
@@ -4,18 +4,15 @@
 
 ### About us
 
-At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will all rely on code and by empowering software developers today, we can bring the future sooner. You can learn more about our mission by reading our [our master plan](https://sourcegraph.com/plan).
+Sourcegraph is on a mission to make it so everyone, in every community, in every country, and in every industry can create products using the best technology. The innovations of the future will all rely on software, so by empowering software developers today we can bring the future sooner. You can learn more about our mission by reading our [our master plan](https://sourcegraph.com/plan).
```

Comment:
I like the idea here, but it seems like there are too many superlatives here ("every ... every ... every ... every ... best"). This makes it too verbose and also too general of a statement. I think a more concise statement like "Sourcegraph's mission is to enable every developer to build technology with the best tools" packs more punch.

Reply:
I lifted this phrasing from the master plan. Do you think we should change it there too?

Reply:
No, I think it is a better fit in that context (leading something called the "master plan" instead of "about us"). Also, the "—not just the ones working at the half-dozen dominant tech companies—" clause in the original serves to make the message more specific here. Without it, the sentence reads like we want to do everything for everyone.